content/applications/hr/recruitment/time_in_stage.rst

```diff
@@ -0,0 +1,82 @@
+======================
+Time in stage analysis
```

Comment:
If this is the name of the report, I would capitalize it in any heading in which it appears.